ivc/src/ivc/mod.rs

```diff
@@ -19,21 +19,28 @@ pub const N_LIMBS_XLARGE: usize = 2;
 /// `test_regression_additional_columns_reduction_to_degree_2`
 pub const N_ADDITIONAL_WIT_COL_QUAD: usize = 335;
 
-/// Number of challenges in the IVC circuit.
-/// It is the maximum number of constraints per row.
-/// We do suppose the Poseidon circuit has the highest number of constraints,
-/// for now.
-/// We also add the number of "blocks" in the IVC circuit as there will be
-/// [N_BLOCKS] alphas required to aggregate all blocks on each row.
-pub const IVC_NB_CHALLENGES: usize = IVC_POSEIDON_NB_CONSTRAINTS + N_BLOCKS;
+/// Number of constraints used by the IVC circuit.
+pub const N_CONSTRAINTS: usize = N_CONSTRAINTS_POSEIDON + N_BLOCKS + 61;
```

Comment:
where does this `+ 61` come from?

Reply:
It's ... the rest of the IVC :) Like EC additions, decomposing inputs, and so on. We could split it down into a sum of 6 other constants, but I see no need for now. Maybe later?

Reply:
Would be good to mention where it comes. Unclear for now.